x-pack/plugins/infra/public/pages/metrics/hosts/components/hosts_table_columns.tsx

```diff
@@ -36,7 +37,6 @@ export const HostsTableColumns: Array<EuiBasicTableColumn<HostNodeRow>> = [
     sortable: true,
     truncateText: true,
     textOnly: true,
-    render: (name: string) => <EuiText size="s">{name}</EuiText>,
```

Comment:
Leaving the rendering up to the table, allows it to correctly handles the text overflow